vivisect/remote/server.py

```diff
@@ -24,7 +24,7 @@
 viv_s_ip = '224.56.56.56'
 viv_s_port = 26998
 
-timeo_wait = 10
+timeo_wait = 1
```

Comment:
Why such a decrease in the queue timeout?

Reply:
this timeout occurs before any data is sent on new connections....  so even with the smallest binaries, there's a 10second delay.
1 sec seemed appropriate, and should still scale to hundreds or thousands of connected clients.

Reply:
The number of connections worries me less than supporting slower connections. 

Also, if the queue is always taking the timeout, that smells like something isn't properly calling `set` to bump the threading.Event that `ChunkQueue` is using. Because timeo_wait is being passed to a `threading.Event`, so it should be the max time we wait, not just a flat sleep.

Reply:
you're not wrong.  i'll dig into it.

Reply:
found it.
the event model around new connections was ad-hoc and terrible.  i think i've implemented a much better solution.